### PR TITLE
Add the UI metadata documentation

### LINKED ISSUE #795: All the (UI) metadata usage should be documented in the directive

### PR DESCRIPTION
**Pull request by sbrunner**
fix  #795

### CODE REVIEW

**Comment by sbrunner** [comment]
fix #795

File: contribs/gmf/src/directives/layertree.js [hunk 1]
```diff
@@ -60,6 +60,14 @@ ngeo.module.value('ngeoLayertreeTemplateUrl',
  * metadata URLs in a new window. By default, and in the default template,
  * links will be opened in a popup.
  *
+ * Used UI metadata:
+ *
+ *  * isChecked: if 'false' the layer visibility will be set to false.
+ *  * legendRule: WMS rule used to get a layer icon.
+ *  * isLegendExpanded: if 'true' the legend is expanded by default
```
Comment (ger-benjamin) [inline contribs/gmf/src/directives/layertree.js]: dot (.) at the end

File: contribs/gmf/src/services/querymanager.js [hunk 2]
```diff
@@ -9,6 +9,14 @@ goog.require('ngeo.Query');
  * The QueryManager service, uses the
  * c2cgeoportal's themes to configure ngeo's query service with each layer
  * found.
+ *
+ * Used UI metadata:
+ *
+ *  * identifierAttributeField: Field used to identify the feature (file a title)
```
Comment (ger-benjamin) [inline contribs/gmf/src/services/querymanager.js]: Should be a dot (.) at the end.

File: contribs/gmf/src/services/querymanager.js [hunk 3]
```diff
@@ -9,6 +9,14 @@ goog.require('ngeo.Query');
...
+ *  * queryLayers: The alternate layers used to do the query.
+ *  * wmsLayers: The layers used to do the query, used if queryLayers is not provided.
+ *  * wmsUrl: An alternate wmsUrl used to do the query (essential to query WMTS layer)
```
Comment (ger-benjamin) [inline contribs/gmf/src/services/querymanager.js]: Should have a dot (.) at the end.

**Comment by ger-benjamin** [comment]
Thanks, please handle my comments then merge.

File: contribs/gmf/src/services/querymanager.js (hunk 2 above)
Comment (sbrunner) [inline contribs/gmf/src/services/querymanager.js]: ??? => like a title

Comment (ger-benjamin) [inline contribs/gmf/src/services/querymanager.js]: better. Or "used as title"

Comment (sbrunner) [inline contribs/gmf/src/services/querymanager.js]: Excellent, thanks :-)

**Pull request merged**